sktime/forecasting/scinet.py

```diff
@@ -0,0 +1,286 @@
+"""Deep Learning Forecaster using SCINet Forecaster."""
+
+from sktime.forecasting.base.adapters._pytorch import BaseDeepNetworkPyTorch
+
+
+class SCINetForecaster(BaseDeepNetworkPyTorch):
+    """SCINet-Linear Forecaster.
+
+    Implementation of the SCINet forecaster.
+
+    Core logic is directly copied from the curelab SCINet implementation
+    at github.
+
+    Parameters
+    ----------
+    seq_len : int
+    Length of the input sequence.
+
+    pred_len : int
+    Length of the prediction (forecast horizon).
+
+    num_epochs : int, default=16
+    Number of epochs to train the model.
+
+    batch_size : int, default=8
+    Number of training examples in each batch.
+
+    in_channels : int, default=1
+    Number of input channels passed to the network.
+
+    individual : bool, default=False
+    Flag that controls whether the network treats each input channel independently.
+
+    If True, a separate linear layer is created for each input channel.
+    If False, a single shared linear layer is used across all channels.
+    criterion : torch.nn Loss Function, default=None
+    Loss function to be used for training. If not provided, a default such as
+    torch.nn.MSELoss is often used.
```

Comment:
@Sohaib-Ahmed21  2 lines from `individual` are still there in the new commit

Reply:
Also can you put indentation with each description of param, so that it is rendered properly, you may see the different between [scinet docs](https://sktime--7400.org.readthedocs.build/en/7400/api_reference/auto_generated/sktime.forecasting.scinet.SCINetForecaster.html) and [cinn docs](https://sktime--7400.org.readthedocs.build/en/7400/api_reference/auto_generated/sktime.forecasting.conditional_invertible_neural_network.CINNForecaster.html), see how the params are rendered

Reply:
Thanks! just fixing.